tutorials/Python/03_Loops/Loops.ipynb

```diff
@@ -103,13 +110,36 @@
     "print('This is the {A} thing, here is the {B}, the {C}, and now the {A} again'.format(A='first', C='third', B='second'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Recent versions of Python (from 3.6 onwards) introduced f-string (format-string,`f\"\"` or `f''`), which are an extremely convenient for string formatting:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"'x' is of type {type(x)} and contains the value {x}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Thus, by prepending our string with the letter `f`, we can use variables and valid Python expressions within `{}`. In the rest of the course we will prefer this new formatting style since it greatly improves readability."
```

Comment:
Might be worth saying that we recommend that all new code uses f-strings (because as you mention better readability) and hence the rest of the course will be using it?

Reply:
> In the rest of the course we will prefer this new formatting style since it greatly improves readability.

Isn't this what you mean? What would you suggest to change it with?

* rest -> reminder

Reply:
Sorry, my poor explanation here. What I meant is, maybe it's worth explicitly saying that we recommend the use of f-strings over other types of formatting?